data/_models/shared/Federated data access.csv

```diff
@@ -0,0 +1,8 @@
+tableName,tableExtends,tableType,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,refLabel,defaultValue,validation,visible,computed,semantics,profiles,label,description
+Federated data access,,,,,,,,,,,,,,,,,,,List of Armadillo and/or Opal server(s) available for this resource.
+Federated data access,,,id,auto_id,1,true,,,,,,,,,,,,,
```

Comment:
I would oppose auto id. Please follow same structure as for all other nested resource properties analogous to subpopulations, datasets, etc. So using composite key (resource,name)

Reply:
Removed auto id, using composite key (resource, name) instead.